.github/build.yaml.gomplate

```diff
@@ -501,5 +479,50 @@ jobs:
           path: |
             *.vhd
           if-no-files-found: error
+
+    {{{ if $config.publishing_pipeline }}}
+
+  publish-vanilla-ami-{{{ $flavor }}}:
+    runs-on: ubuntu-latest
+    needs: [raw-img-{{{ $flavor }}}, tests-squashfs-{{{ $flavor }}}]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download result for build
+        uses: actions/download-artifact@v2
+        with:
+          name: cOS-raw-img-{{{ $flavor }}}
+          path: .
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+      - name: Upload and publish vanilla image
+        run: |
+          make aws_vanilla_ami
+
+  ami-publish-{{{ $flavor }}}:
```

Comment:
Currently this will only upload the suse image.

Reply:
Correct